packages/models-library/src/models_library/projects_access.py

```diff
@@ -30,18 +28,15 @@ class AccessRights(BaseModel):
 
 
 class Owner(BaseModel):
-    user_id: PositiveInt = Field(..., description="Owner's user id")
-    first_name: FirstNameStr | None = Field(..., description="Owner's first name")
-    last_name: LastNameStr | None = Field(..., description="Owner's last name")
+    user_id: UserID = Field(..., description="Owner's user id")
```

Comment:
`Annotated[UserID, ...` can use prompt to change this

Reply:
I am a bit scarred of this.